### DISCUSSION TITLE
Trying whitelabeling RC and cannot compile for ios...

### DISCUSSION THREAD

**Discussion by slayderx3** [RocketChat/Rocket.Chat.ReactNative#5932] [General]
Trying to whitelabel the RC.Standalone and getting some unreal problem )
At first:
Xcode Version 16.0 (16A242d) crashes when trying to compile project... Found a problem with this dependency
react-native-image-crop-picker
(https://stackoverflow.com/questions/78995007/xcode-16-0-unexpected-service-error-the-xcode-build-system-has-crashed)

Updated it to version 0.41.2
yarn upgrade react-native-image-crop-picker@0.41.2

Now when trying to build have some problems with FMDB... It is not provided now to install via npm and when trying to install node_modules getting error (yarn_error.log):
  Error: https://registry.yarnpkg.com/@fmdb: Request "https://registry.yarnpkg.com/@fmdb" returned a 405
  
Compilation starts but failing to find FMDB.h in public headers folder in ios/pods...

Now trying to copy them there (also DatabaseDeleteHelper.h) to Headers folder and getting other problems...

Rocket.Chat.ReactNative/ios/Shared/RocketChat/Database.swift:21:34: Value of type 'Database' has no member 'queryRaw'

I'm stuck. Can anyone help?

**Comment by slayderx3** [comment]
Also tried to include FMDB as library in podfile but have another errror with duplicating code on link stage...

**Comment by diegolmello** [comment]
https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/5877
